{"comments": {}, "bugs": {"645269": {"comments": [{"id": 5371442, "raw_text": "E.g.,\n\n1301111546636\tEngine.History\tWARN\tError creating record: Key bundle is null for undefined JS Stack trace: encrypt()@record.js:212 < SyncEngine__uploadOutgoing()@engines.js:924 < SyncEngine__sync()@engines.js:991 < wrappedSync(null)@util.js:193 < batchedSync()@util.js:199 < WrappedNotify()@util.js:172 < Engine_sync()@engines.js:388 < WeaveSvc__syncEngine([object Object])@service.js:1913 < ()@service.js:1793 < WrappedNotify()@util.js:172 < WrappedLock()@util.js:140 < _lockedSync()@service.js:1695 < ()@service.js:1686 < WrappedCatch()@util.js:114 < sync()@service.js:1667 < (3)@browser.js:5317\n\n...\n\n1301111547003\tCollection\tDEBUG\tPOST fail 401 https://scl2-sync609.services.mozilla.com/1.0/wdicxnohk5he76oalkh73pzvmpdyfaej/storage/history\n1301111547004\tService.Main\tDEBUG\tFinding cluster for user \n1301111547401\tNet.Resource\tDEBUG\tGET fail 400 https://auth.services.mozilla.com/user/1.0//node/weave\n1301111547402\tEngine.History\tDEBUG\tUploading records failed: \"Authentication failed\"\n1301111547600\tService.Main\tDEBUG\tFinding cluster for user", "creation_time": "2011-03-26T03:53:24Z", "text": "E.g.,\n\n1301111546636\tEngine.History\tWARN\tError creating record: Key bundle is null for undefined JS Stack trace: encrypt()@record.js:212 < SyncEngine__uploadOutgoing()@engines.js:924 < SyncEngine__sync()@engines.js:991 < wrappedSync(null)@util.js:193 < batchedSync()@util.js:199 < WrappedNotify()@util.js:172 < Engine_sync()@engines.js:388 < WeaveSvc__syncEngine([object Object])@service.js:1913 < ()@service.js:1793 < WrappedNotify()@util.js:172 < WrappedLock()@util.js:140 < _lockedSync()@service.js:1695 < ()@service.js:1686 < WrappedCatch()@util.js:114 < sync()@service.js:1667 < (3)@browser.js:5317\n\n...\n\n1301111547003\tCollection\tDEBUG\tPOST fail 401 https://scl2-sync609.services.mozilla.com/1.0/wdicxnohk5he76oalkh73pzvmpdyfaej/storage/history\n1301111547004\tService.Main\tDEBUG\tFinding cluster for user \n1301111547401\tNet.Resource\tDEBUG\tGET fail 400 https://auth.services.mozilla.com/user/1.0//node/weave\n1301111547402\tEngine.History\tDEBUG\tUploading records failed: \"Authentication failed\"\n1301111547600\tService.Main\tDEBUG\tFinding cluster for user", "count": 0, "tags": [], "creator": "bugzilla@twinql.com", "bug_id": 645269, "author": "bugzilla@twinql.com", "is_private": false, "time": "2011-03-26T03:53:24Z", "attachment_id": null}, {"author": "philipp@weitershausen.de", "bug_id": 645269, "creator": "philipp@weitershausen.de", "count": 1, "tags": [], "text": "We should probably lock Service.startOver() with the same lock that sync() is locked with.", "creation_time": "2011-03-26T09:23:29Z", "raw_text": "We should probably lock Service.startOver() with the same lock that sync() is locked with.", "id": 5371622, "attachment_id": null, "time": "2011-03-26T09:23:29Z", "is_private": false}]}}}